pkg/settlement/swap/transaction/transaction_test.go

```diff
@@ -27,6 +27,16 @@ func nonceKey(sender common.Address) string {
 	return fmt.Sprintf("transaction_nonce_%x", sender)
 }
 
+func wrapGasprice(gasprice *big.Int) *big.Int {
+	return transaction.IncreaseGasPrice(gasprice)
+}
```

Comment:
I think you can have a mock for `IncreaseGasPrice`

Reply:
I d't known what's your mean?

Reply:
https://betterprogramming.pub/a-gomock-quick-start-guide-71bee4b3a6f1

What I meant was instead of directly calling an irrelevant method you can make up the output of that method call. However, I believe whatever the logic is in `IncreaseGasPrice` is relevant but you are introducing more trouble by making extra methods.